### PR TITLE
Configure autoscaling profile in GKE clusters

### PR DESCRIPTION
**Pull request by chrissng**
Configure autoscaling profile in GKE clusters (https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#autoscaling_profiles)

TODOs:
- [x] Implement autoscaling profile argument in `google_container_cluster` resource.
- [x] Implement tests
- [x] Update terraform documentation

**Release Note Template for Downstream PRs (will be copied)**

```release-note:REPLACEME
Configure autoscaling profile in GKE clusters (https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#autoscaling_profiles)
```

### CODE REVIEW

**Comment by danawillow** [comment]
Hey @chrissng, as you can see from https://github.com/GoogleCloudPlatform/magic-modules/pull/3076#issuecomment-582340194 we try to randomly assign reviewers for external PRs so we don't overload single members of the team. I'm going to go ahead and remove myself, but feel free to tag me back in if you run into any issues.

Also while I'm here, if your PR fixes an open issue that's filed in TPG, please make sure your PR description contains the phrase "fixes [link to issue]" and that you've claimed the issue there by commenting on it so nobody else tries to do it at the same time. Thanks!

File: third_party/terraform/tests/resource_container_cluster_test.go.erb [hunk 1]
```diff
@@ -2819,6 +2865,24 @@ resource "google_container_cluster" "with_node_pool" {
...
 
+<% unless version == 'ga' -%>
+func testAccContainerCluster_withAutoscalingProfile(cluster string, autoscalingProfile string) string {
+	config := fmt.Sprintf(`
+resource "google_container_cluster" "autoscaling_with_profile" {
+  name               = "%s"
+  location           = "us-central1-a"
+  initial_node_count = 1
+
+	cluster_autoscaling {
```
Comment (emilymye) [inline third_party/terraform/tests/resource_container_cluster_test.go.erb]: nit - tab/spaces

**Comment by emilymye** [comment]
Wow, great work! The tests seem to pass so I think this is actually just ready! I have one nit comment, but if you don't get to it by later today we can just merge and followup fix in a different PR

File: third_party/terraform/tests/resource_container_cluster_test.go.erb (hunk 1 above)
Comment (chrissng) [reply third_party/terraform/tests/resource_container_cluster_test.go.erb]: Fixed. Thanks!

**Pull request merged**